docs/developer/advanced/upgrading-nodejs.asciidoc

```diff
@@ -37,6 +43,13 @@ Due to Node.js 16 coming to an https://nodejs.org/en/blog/announcements/nodejs16
 To keep support for these older platforms, we're bundling the Linux distributable of {kib} with a https://github.com/elastic/kibana-custom-nodejs-builds[custom build of Node.js] with extended backwards compatibility.
 The only difference between the offical Node.js build and our custom build, is the version of `glibc` that it's compiled against.
 
+[[start-new-nodejs-build]]
+==== How to start a new build
+
+To generate a new custom Node.js build, start the "https://buildkite.com/elastic/kibana-custom-node-dot-js-builds#new[custom node.js builds]" Buildkite pipeline.
```

Comment:
This link isn't rendering properly. Also it is a private pipeline, does it need to be?

Reply:
Good question. Not really, [the repo](https://github.com/elastic/kibana-custom-nodejs-builds) in which this pipeline is defined is public. But I'm not sure if there's environment variables or something along those lines in the Buildkite pipeline which should be kept private?

Reply:
I looked through the env variables and didn't find any sensitive information, but I'm not sure all the implications of making the pipeline public. Adding "(only accessible by Elastic employees)" as mentioned below seems good enough for now.